Source/UserInterface.cs

```diff
@@ -676,7 +677,12 @@ public string FormatTimeUntilDecayInSecondsToString(double TimeUntilDecayInSecon
             {
                 DecayTimeString = "Error!";
             }
-            
+            catch (OverflowException)

```

Comment:
it was fine on new games, in old game with loads of ships it was spaming with it